packages/ui-expandable/src/Expandable/props.ts

```diff
@@ -105,5 +105,10 @@ const allowedProps: AllowedPropKeys = [
   'render'
 ]
 
-export type { ExpandableProps, ExpandableState, RenderExpandable }
+export type {
+  ExpandableProps,
+  ExpandableState,
+  RenderExpandable,
+  GetToggleProps
```

Comment:
Sry, I meant you should rename `GetToggleProps` to `ExpandableToggleProps`, because if we export something from a package, the variable name should somehow include the component's name.